digfort.rb

```diff
@@ -42,6 +47,8 @@
 
 =end
 
+puts "The digfort script is deprecated. Please use DFHack's quickfort command instead. The digfort script will be removed in a future DFHack release."
```

Comment:
Can this mention "quickfort run"? I'm not sure how easy that would be to work in - I see it's in the comment above, but I don't think there's a way to easily view that from the DFHack console.

Maybe something simple like this could work. It would be cool if the filename could be included somehow too, but since it would need to be moved to the `blueprints` folder, that might end up causing more confusion.

```suggestion
puts "The digfort script is deprecated. Please use DFHack's quickfort command instead:\n`quickfort run example.csv`, where example.csv is in the blueprints folder.\nThe digfort script will be removed in a future DFHack release."
```

Reply:
reworded a bit. I'll update the fortplan deprecation notice similarly.